report-viewer/src/model/Language.ts

```diff
@@ -13,6 +13,7 @@ enum ParserLanguage {
   EMF_METAMODEL = 'EMF metamodel',
   EMF_MODEL = 'EMF models (dynamically created token set)',
   GO = 'Go Parser',
+  JAVA_CPG = 'Java CPG Module'
```

Comment:
Missing comma breaks the build:
```suggestion
  JAVA_CPG = 'Java CPG Module',
```

Reply:
Sorry, that slipped right by me.
Sonar seems to fail if there is nothing in the commit that it can identify as code.